README.md

```diff
@@ -15,15 +15,17 @@ Welcome to [Data Together](https://datatogether.org/), we're building a better f
 
 ## Get Involved
 
-If you would like to **add to the discussion**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md). 
+If you would like to **add to the discussion**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md).
 
 If you want to **get into code and start hacking**, check out [first timer](https://github.com/issues?q=is:open+org:datatogether+label:first-timers-only) issues! [Webapp](https://github.com/datatogether/webapp) and [sentry](http://github.com/datatogether/sentry) are the best points of entry. We track project progress using a [Waffle Board](https://waffle.io/datatogether/roadmap) and identify major goals in our [Roadmap repository](https://github.com/datatogether/roadmap/).
 
+If you would like to get **email notification** , please join our [public google group] (https://groups.google.com/forum/#!forum/datatogether/join)
```

Comment:
Please add an `s` at the end of `notification` so that it's plural. They'll be receiving more than one email if they join =). Also, remove the `space` before the comma here.

Reply:
Please remove space between the link text `[public google group]` and the link `(https://....)` so that it formats correctly.